api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -168,6 +168,10 @@ message RedisProxy {
         bool exclude_read_commands = 3;
       }
 
+      message ReadCommandPolicy {
+        string cluster = 1 [(validate.rules).string = {min_len: 1}];
```

Comment:
Please add comments describing the new fields.

Reply:
fixed